README.md

```diff
@@ -47,7 +47,7 @@ By default, if no selector parameter is provided, the `runSelector` step selects
 upstream job.
 
 ```groovy
-def runWrapper = runSelector job: 'upstream-project-name'
+def runWrapper = runSelector 'upstream-project-name'
```

Comment:
A single `@DataBoundConstructor` (i.e., mandatory) parameter need not be specified.